mix.exs

```diff
@@ -54,7 +54,7 @@ defmodule Elasticsearch.Mixfile do
       {:jason, ">= 0.0.0", optional: true},
       {:telemetry, "~> 0.4.3 or ~> 1.0"},
       {:vex, "~> 0.6"},
-      {:car_req, github: "carsdotcom/car_req", overrride: true},
```

Comment:
don't we want to not have a ref so this lib as a dependency can update when we change car_req? Maybe we want to ditch `override` instead?

Reply:
You are right , should only remove the override

Reply:
Done